packages/where/index.js

```diff
@@ -24,28 +24,25 @@ const whereEndoStateWindows = env => {
   if (env.HOMEDRIVE !== undefined && env.HOMEPATH !== undefined) {
     return `${env.HOMEDRIVE}${env.HOMEPATH}\\AppData\\Endo`;
   }
-  return '.';
+  return 'Endo';
 };
 
 /**
  * @type {typeof import('./types.js').whereEndoState}
  */
 export const whereEndoState = (platform, env) => {
-  if (platform === 'win32') {
-    return whereEndoStateWindows(env);
+  if (env.XDG_STATE_HOME !== undefined) {
```

Comment:
The name `XDG_STATE_HOME` is not chosen by this package, right? It's intended to conform to [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) ? Please cite the external constraint.